source-map-support.js

```diff
@@ -18,7 +21,13 @@ var sourceMapCache = {};
 var reSourceMap = /^data:application\/json[^,]+base64,/;
 
 function isInBrowser() {
-  return ((typeof window !== 'undefined') && (typeof XMLHttpRequest === 'function'));
+  if (environment === "browser")
+    return true;
+  if (environement === "node")
```

Comment:
No, no I don't think so. =/
It should be defined in line 12?

But I did forget to support changing the environment via the options
object. Whoops.

@tugend

2015-10-19 17:12 GMT+02:00 Asger Hallas <notifications@github.com>:

> In source-map-support.js
> <https://github.com/evanw/node-source-map-support/pull/105#discussion_r42382922>
> :
>
> > @@ -18,7 +21,13 @@ var sourceMapCache = {};
> >  var reSourceMap = /^data:application\/json[^,]+base64,/;
> >
> >  function isInBrowser() {
> > -  return ((typeof window !== 'undefined') && (typeof XMLHttpRequest === 'function'));
> > +  if (environment === "browser")
> > +    return true;
> > +  if (environement === "node")
>
> @tugend <https://github.com/tugend> environement is not defined
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/evanw/node-source-map-support/pull/105/files#r42382922>
> .
>